atlas/resource_atlas_schema.go

```diff
@@ -41,6 +43,61 @@ func newSchemaResource() *schema.Resource {
 	}
 }
 
+func customizeDiff(ctx context.Context, diff *schema.ResourceDiff, i interface{}) error {
+	oldV, newV := diff.GetChange("hcl")
+	if oldV == nil {
+		return nil
+	}
+	s, ok := oldV.(string)
+	if !ok || s != "" {
+		return nil
+	}
```

Comment:
I think I am missing context here, why is this "okay"?

Reply:
See line 92. I want this check only on the first execution. i.e. when there is no previous state. Makes sense?